native/packages/react-native-bpk-component-picker/src/common-types.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import { type Node } from 'react';
+import PropTypes from 'prop-types';
+
+export type PickerValue = string | number;
```

Comment:
Why do we support both string and number?

Reply:
We're matching what the underlying Picker component uses: https://github.com/facebook/react-native/blob/master/Libraries/Components/Picker/Picker.js#L53

(It says `any` but then right above it, says string or number, so I went with the stricter one).